stratum/hal/lib/common/p4_service.cc

```diff
@@ -215,7 +215,9 @@ void LogWriteRequest(uint64 node_id, const ::p4::v1::WriteRequest& req,
   if (FLAGS_write_req_log_file.empty()) {
     return;
   }
-  if (results.size() != req.updates_size()) {
+  if (results.empty()) {
+
+  } else if (results.size() != req.updates_size()) {
```

Comment:
How about
```suggestion
if (!results.empty() && results.size() != req.updates_size()) {
```

Reply:
That was just a placeholder. I have not thought about the solution yet, I guess yours could work.